src/linker/cli.py

```diff
@@ -129,18 +129,15 @@ def run_slurm_job(
     when a slurm computing environment is defined in the environment.yaml.
     """
     configure_logging_to_terminal(verbose)
-    results_dir = Path(results_dir)
-    implementation_dir = Path(implementation_dir)
-    input_data = [Path(x) for x in input_data]
     main = handle_exceptions(
         func=runner.run_container, exceptions_logger=logger, with_debugger=False
     )
     main(
         container_engine=container_engine,
-        input_data=input_data,
-        results_dir=results_dir,
+        input_data=[Path(x) for x in input_data],
```

Comment:
What's the reason for this refactor? Just simplicity?

Reply:
Yes, and to get rid of some linting warnings (that, frankly, confuse me).

Implemented in followup PR